db/people.yml

```diff
@@ -539,3 +539,30 @@ sbanerjee:
   name: Soham Banerjee
   orcid_id: 0000-0001-9271-493X
   position: Graduate Student
+  orcid_id: 0000-0001-9271-493X
```

Comment:
Did you inadvertently add a line here?

Reply:
Yes, I add this line inadvertently maybe during rebasing. Should I further commit on this branch or start another branch?

Reply:
simply delete the line and then push again.  Then I can merge